dataactbroker/scripts/initialize.py

```diff
@@ -53,17 +53,13 @@ def createAdmin():
     """Create initial admin user."""
     adminEmail = CONFIG_BROKER['admin_email']
     adminPass = CONFIG_BROKER['admin_password']
-    userDb = UserHandler()
-    try:
-        user = userDb.getUserByEmail(adminEmail)
-    except ResponseException as e:
-
-        if type(e.wrappedException) is NoResultFound:
-            userDb.createUserWithPassword(
-                adminEmail, adminPass, Bcrypt(), permission=2)
-            user = userDb.getUserByEmail(adminEmail)
-            userDb.addUserInfo(user, "Admin", "SYS", "System Admin")
-    userDb.close()
+    with databaseSession() as sess:
+        user = sess.query(User).filter(User.email==adminEmail).one_or_none()
```

Comment:
Never mind! There's no need for a case insensitive check here since the email comes from the config file.